src/main/markers/waveform.segments.js

```diff
@@ -32,13 +32,17 @@ define([
       var segmentGroups = [{ group: segmentZoomGroup, view: 'zoomview' }, { group: segmentOverviewGroup, view: 'overview' }];
 
       var menter = function (event) {
-        this.parent.label.show();
-        this.parent.view.segmentLayer.draw();
+        if (this.parent && this.parent.label)
+          this.parent.label.show();
+        if (this.parent && this.parent.view)
+          this.parent.view.segmentLayer.draw();
```

Comment:
Put the one-line if on the same line - i.e., `if (thing) otherThing`, or use `{}` brackets.